frontend/src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,35 @@
+import NextAuth from "next-auth"
+import SpotifyProvider from "next-auth/providers/spotify"
+
+const scope =
+  "user-read-recently-played user-read-playback-state user-top-read user-modify-playback-state user-read-currently-playing user-follow-read playlist-read-private user-read-email playlist-modify-private playlist-modify-public playlist-read-private user-read-private user-library-read playlist-read-collaborative"
```

Comment:
Any reason why we need to use that many scopes? We only require `playlist-modify-private playlist-modify-public playlist-read-private`.

Reply:
It's true, I change it 👌